specification/attributes/key_value_format.md

```diff
@@ -0,0 +1,32 @@
+# Key-Value Format
+
+Columns that provide Key-Value information are often used in place of separate columns for enumerating data which would be inherently sparse and/or without predetermined keys. This consolidates related information and provides more consistency in the schema. Key-value pairs are also referred to as name-value pairs, attribute-value pairs, or field-value pairs.
+
+All key-value related columns defined in the FOCUS specification MUST follow the key-value formatting requirements listed below. These columns SHALL NOT contain nested elements.
```

Comment:
Is the last sentence needed here? It's one of the requirements listed in the requirements already.

Reply:
No reason to duplicate

Reply:
It was intended to make this more approachable, but the common opinion is that this redundant comment doesn't add value.